lambda-notes/src/views/NoteView.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Route, Switch } from 'react-router-dom';
+import EditView from './EditView';
+import {
+  fetchNoteById,
+  deleteNote
+} from '../actions';
+import { Note, Sidebar, DeleteModal } from '../components';
+
+class NoteView extends React.Component {
+
+  componentDidMount() {
+    const noteId = this.props.match.params.id;
+    this.props.fetchNoteById(noteId);
+  }
+
+
+  render() {
+    return(
+      <div className='container'>
+        <Sidebar />
+        <Switch>
+          <Route exact path='/:id/edit' render={(props) => (<EditView {...props} /> )}/>
+            <Route path='/:id' render={(props) => (
+              <Note {...props} note={this.props.note} />
+            )} />
+
+          </Switch>
+          <Route exact path='/:id/delete' render={(props) => (
+            <div className='modal'><DeleteModal {...props} deleteNote={this.props.deleteNote} />
+          </div>)} />
+
+      </div>
+
+    )
+  }
+}
+
+
+
+const mapStateToProps = state => {
```

Comment:
```suggestion
const mapStateToProps = ({ notesReducer, noteReducer }) => ({
  noteUpdated: notesReducer.noteUpdated,
  note: noteReducer.note,
  error: noteReducer.error,
  fetchingNote: noteReducer.fetchingNote
})
```
Look into destructuring, youll find many ways to reduce boilerPlate cide